lib/studies/reply/inbox.dart

```diff
@@ -1,12 +1,166 @@
 import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+import 'package:gallery/studies/reply/responsive_widget.dart';
 
 class InboxPage extends StatelessWidget {
   const InboxPage();
 
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(child: Text('Hello World')),
+    return AdaptiveNav();
+  }
+}
+
+class AdaptiveNav extends StatefulWidget {
+  AdaptiveNav({Key key}) : super(key: key);
+
+  @override
+  _AdaptiveNavState createState() => _AdaptiveNavState();
+}
+
+class _AdaptiveNavState extends State<AdaptiveNav> {
+  int _selectedIndex = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    final localizations = GalleryLocalizations.of(context);
+
+    final _navigationItems = <String, IconData>{
+      localizations.replyInboxLabel: Icons.inbox,
+      localizations.replyStarredLabel: Icons.star_border,
+      localizations.replySentLabel: Icons.send,
+      localizations.replyTrashLabel: Icons.delete_outline,
+      localizations.replySpamLabel: Icons.error_outline,
+      localizations.replyDraftsLabel: Icons.drafts,
+    };
+
+    return ResponsiveWidget(
+      desktopScreen: _BuildDesktopNav(
+        selectedIndex: _selectedIndex,
+        extended: true,
+        destinations: _navigationItems,
+        onItemTapped: _onItemTapped,
+      ),
+      tabletScreen: _BuildDesktopNav(
+        selectedIndex: _selectedIndex,
+        extended: false,
+        destinations: _navigationItems,
+        onItemTapped: _onItemTapped,
+      ),
+      mobileScreen: _BuildMobileNav(
+        selectedIndex: _selectedIndex,
+        destinations: _navigationItems,
+        onItemTapped: _onItemTapped,
+      ),
+    );
+  }
+
+  void _onItemTapped(int index) {
```

Comment:
_onItemSelected works better with a mix of taps and clicks

Reply:
maybe onDestinationSelected?

Reply:
I'll try that